.github/workflows/docs.yml

```diff
@@ -0,0 +1,47 @@
+
+name: Documentation
+on:
+  push:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        
+      - name: Prepare apt repositories
+        run: | 
+          sudo sed -i '/deb-src/s/^# //' /etc/apt/sources.list
+          sudo apt update
+
+      - name: Install sugar-toolkit-gtk3
+        run: |
+          # enable deb-src repositories
+          
+          sudo apt build-dep sugar-toolkit-gtk3
+          sudo apt install python-is-python3 python3-cairo python3-gi python3-gi-cairo python3-dateutil python3-decorator \
+            gir1.2-telepathyglib-0.12 gir1.2-webkit2-4.0 python3-empy gir1.2-sugarext-1.0
+          ./autogen.sh --with-python2
+          make
+          ./autogen.sh --with-python3
+          make
+          
+      - name: Install Sphinx
+        run: |
+          sudo apt-get -qqy install sphinx-common
```

Comment:
apt-get -> apt

Reply:
`apt` does not have a stable CLI interface. See [this askubuntu post](https://askubuntu.com/questions/990823/apt-gives-unstable-cli-interface-warning)

Reply:
Should use `apt-get` as `apt` is an end-user tool per `apt(8)`.  But change is needed, because your use is inconsistent; you are using `apt` for build and runtime dependencies instead of `apt-get`.